docs-content/sdk/client.md

```diff
@@ -245,10 +245,91 @@ slug: client
   </div>
 </section>
 
+<section className="section">
+  <div className="left">
+    <h3>H.startSpan</h3>
```

Comment:
one thing i noticed with the `startSpan` and `startManualSpan` API is that it doesn't allow returning the value of the callback in some cases. should we make it clear that returning the return value of the `cb` is / is not an option?

Reply:
Good callout, @Vadman97! I ended up fixing that in https://github.com/highlight/highlight/pull/9335/commits/984435d30f5529a92c9dc2c694a86d59986c9ce0. Mind giving those changes a quick look? Will discard your previous review - thanks for looking this over!

Reply:
reviewed!